### PR TITLE
Make sure `auth` is included in `collection.toJSON`

### PR DESCRIPTION
**Pull request by kamalaknn**

### CODE REVIEW

**Comment by kamalaknn** [comment]
@SamvelRaja this should fix your troubles

File: test/unit/collection.test.js [hunk 1]
```diff
@@ -143,19 +143,72 @@ describe('Collection', function () {
...
+            // version info does not exactly have a one to one mapping
+            // with the definition and toJSON, hence the hack
+            expect(collectionJSON.info.version).to.have.property('string', '2.1.0');
+            delete collectionJSON.info.version;
+            delete collectionDefinition.info.version;
+
+            expect(collectionJSON).to.eql(collectionDefinition);
+
+            // check for root level properties moved to info
+            expect(collectionJSON).to.not.have.property('id');
```
Comment (harryi3t) [inline test/unit/collection.test.js]: Why extra checks?
```js
expect(collectionJSON).to.eql(collectionDefinition);
```
Doesn't this takes care of everything?

**Comment by harryi3t** [comment]
@kamalaknn changelog needs to be updated with the bug fix entry?

File: test/unit/collection.test.js (hunk 1 above)
Comment (kamalaknn) [reply test/unit/collection.test.js]: Check the comment above

**Pull request merged**